components/index/CardsMonitoring/MonitoringItemsOverview/Card.vue

```diff
@@ -61,14 +61,22 @@
             :items="monitoringItems"
           />
         </section>
-        <div>
+        <div :class="$style.buttonWrap">
```

Comment:
```suggestion
        <div :class="$style['button-wrap']">
```

何度もすみません...！nuxt generateにおいて、css modulesの命名でcamelCaseは効かないみたいなのです。こちら↑のようにkebab-caseにしていただけるとよいかと思います 🙏

Reply:
確認不足でした、すみません🙇